scienceapi/manage.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "scienceapi.settings.local")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings.local")
```

Comment:
Undo this change and move the `manage.py` file to the newly created `/app` directory